--- meson.build	2024-06-26 20:58:26.753005180 -0500
+++ _meson.build	2024-07-19 20:14:50.876736109 -0500
@@ -8,17 +8,8 @@
 
 uname_output = run_command('uname', check: true).stdout().strip()
 if uname_output == 'Linux'
-  config_h = configuration_data()
-  pci_dep = dependency('libpci', required: false)
-  if pci_dep.found()
-    link_args += ['-lpci']
-    config_h.set('PCI_DETECTION', 1)
-  else
-   config_h.set('PCI_DETECTION', 0)
-  endif
-  configure_file(output: 'config.h', configuration: config_h)
   c_args += ['-Os', '-s', '-fomit-frame-pointer', '-fno-unwind-tables', '-fno-asynchronous-unwind-tables', '-g0']
-  executable('tinyfetch', 'src/tinyfetch.c', install : true, c_args: c_args, link_args: link_args)
+  executable('tinyfetch', 'src/tinyfetch.c', install : true, c_args: c_args)
 elif uname_output == 'FreeBSD'
   config_h = configuration_data()
   pci_dep = dependency('libpci', required: false)
